j2k/testData/fileOrElement/switch/caseWithDefaultblock.java

```diff
@@ -0,0 +1,21 @@
+//method
+void foo() {
```

Comment:
You don't need a `main` method but you need a file that can be highlighted without errors in the editor.